web/ui/src/lib/clustersMap/campus/angouleme.ts

```diff
@@ -0,0 +1,63 @@
+import { Campus } from '../campus';
+import { Cluster } from '../cluster';
+import { ICampus } from '../types';
+
+//
+export class Angouleme extends Campus implements ICampus {
+  emoji = (): string => '🇫🇷';
+
+  name = (): string => 'Angouleme';
+
+  extractorRegexp = (): RegExp =>
+    /(?<clusterWithLetter>c(?<cluster>\d+))(?<rowWithLetter>r(?<row>\d+))(?<workspaceWithLetter>p(?<workspace>\d+))/i;
```

Comment:
This regexp cannot works, this regexp works for identifier like `c1r2p3` not `1A2` 

```suggestion
    /(?<clusterWithLetter>(?<cluster>\d+))(?<rowWithLetter>(?<row>\w))(?<workspaceWithLetter>(?<workspace>\d+))/i;
```